Backend/app/rider-platform/rider-app/Main/spec/Storage/PickupInstructions.yaml

```
@@ -59,4 +59,13 @@
         order: asc
-      limit: 1
+      limit: 1
+
+    deleteInstructionById:
+      kvFunction: deleteWithKV
+      where: id
+
+    updateMediaFileById:
+      kvFunction: updateWithKV
+      params: [mediaFileId]
+      where: id
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)